hexagon/support/update_hexagon.py

```diff
@@ -71,11 +72,12 @@ def check_for_hexagon_updates():
     if not inquirer.confirm("Would you like to update?", default=True).execute():
         return
 
-    subprocess.check_call(
-        f"{sys.executable} -m pip install https://github.com/{REPO_ORG}/{REPO_NAME}/releases/download/v{latest_github_release_version}/hexagon-{latest_github_release_version}.tar.gz",
-        shell=True,
-        stdout=subprocess.DEVNULL,
-    )
+    with Halo(text="Updating"):
+        subprocess.check_call(
+            f"{sys.executable} -m pip --disable-pip-version-check install https://github.com/{REPO_ORG}/{REPO_NAME}/releases/download/v{latest_github_release_version}/hexagon-{latest_github_release_version}.tar.gz",
+            shell=True,
+            stdout=subprocess.DEVNULL,
+        )
     log.info("[green]✔️ [white]Updated to latest version")
```

Comment:
```suggestion
    log.info("[green]🗸 [white]Updated to latest version")
```
esto es bien de obse

Reply:
http://xahlee.info/comp/unicode_punctuation_symbols.html